### ISSUE TITLE
As a Commons visitor, I encounter an error for a site's missing file.

### ISSUE THREAD

**Issue by kimisgold** [omeka/plugin-Sites#4]
As a visitor to the Commons, I click on the "Sites" tab and receive the following error:

**InvalidArgumentException
Could not find file 433452-M.jpg!**

```
#0 /Users/kim/Sites/omeka-commons/application/libraries/globals.php(1206): web_path_to('433452-M.jpg')
#1 /Users/kim/Sites/omeka-commons/plugins/Sites/helpers/functions.php(74): src('433452-M.jpg')
#2 /Users/kim/Sites/omeka-commons/plugins/Sites/views/public/index/browse.php(10): sites_site_logo(Object(Site))
#3 /Users/kim/Sites/omeka-commons/application/libraries/Omeka/View.php(117): include('/Users/kim/Site...')
#4 /Users/kim/Sites/omeka-commons/application/libraries/Zend/View/Abstract.php(888): Omeka_View->_run('/Users/kim/Site...')
#5 /Users/kim/Sites/omeka-commons/application/libraries/Zend/Controller/Action/Helper/ViewRenderer.php(900): Zend_View_Abstract->render('index/browse.ph...')
#6 /Users/kim/Sites/omeka-commons/application/libraries/Zend/Controller/Action/Helper/ViewRenderer.php(921): Zend_Controller_Action_Helper_ViewRenderer->renderScript('index/browse.ph...', NULL)
#7 /Users/kim/Sites/omeka-commons/application/libraries/Zend/Controller/Action/Helper/ViewRenderer.php(960): Zend_Controller_Action_Helper_ViewRenderer->render()
#8 /Users/kim/Sites/omeka-commons/application/libraries/Zend/Controller/Action/HelperBroker.php(277): Zend_Controller_Action_Helper_ViewRenderer->postDispatch()
#9 /Users/kim/Sites/omeka-commons/application/libraries/Zend/Controller/Action.php(527): Zend_Controller_Action_HelperBroker->notifyPostDispatch()
#10 /Users/kim/Sites/omeka-commons/application/libraries/Zend/Controller/Dispatcher/Standard.php(308): Zend_Controller_Action->dispatch('browseAction')
#11 /Users/kim/Sites/omeka-commons/application/libraries/Zend/Controller/Front.php(954): Zend_Controller_Dispatcher_Standard->dispatch(Object(Zend_Controller_Request_Http), Object(Zend_Controller_Response_Http))
#12 /Users/kim/Sites/omeka-commons/application/libraries/Zend/Application/Bootstrap/Bootstrap.php(97): Zend_Controller_Front->dispatch()
#13 /Users/kim/Sites/omeka-commons/application/libraries/Zend/Application.php(366): Zend_Application_Bootstrap_Bootstrap->run()
#14 /Users/kim/Sites/omeka-commons/application/libraries/Omeka/Application.php(79): Zend_Application->run()
#15 /Users/kim/Sites/omeka-commons/index.php(23): Omeka_Application->run()
#16 {main}
```

Prior to this, I as a Commons site owner and contributor uploaded a logo image from the Commons plugin on my site. The "Settings" view in my Omeka Commons plugin shows my uploaded logo properly. 

As an admin of the Omeka Commons, I go to investigate the edit view of the site within the Sites section of the admin interface. I get the same error as above.

**Comment by patrickmj** [comment]
Should be fix in latest

**Comment by kimisgold** [comment]
As a Commons contributor, I successfully uploaded an avatar via the "Settings" interface in my admin interface. However, I then uploaded a new image to replace my old avatar. The image appears in my own settings, but when going to check on it in the Omeka Commons sites section, I receive an error that my logo file could not be found.

From the view of the Omeka Commons administrator, the 'commons_settings' in the 'omeka_sites' table appears to be updated, but 'Sites/views/shared/images/$site_id' does not have the new file.
